apps/portals/src/configurations/adknowledgeportal/routeControlWrapperProps.ts

```diff
@@ -1,19 +1,45 @@
 import { RouteControlWrapperProps } from 'portal-components/RouteControlWrapper'
-import { SynapseConfig } from 'types/portal-config'
+import {
+  data,
+  people,
+  programs,
+  projects,
+  publications,
+  studies,
+} from './synapseConfigs'
+import { programsSql } from './resources'
+import experimentalTools from './synapseConfigs/experimental_tools'
+import computationalTools from './synapseConfigs/computational_tools'
+import targetEnablingResources from './synapseConfigs/target_enabling_resources'
+import { results } from './synapseConfigs/results'
 
 const routeControlWrapper: RouteControlWrapperProps = {
-  synapseConfig: {} as SynapseConfig,
   customRoutes: [
-    'Programs',
-    'Projects',
-    'Studies',
-    'Data',
-    'Publications',
-    'People',
-    'Experimental Models',
-    'Computational Tools',
-    'Target Enabling Resources',
-    'Results',
+    {
+      path: 'Programs',
+      synapseConfigArray: [
+        {
+          className: 'ProgramCardList',
+          name: 'CardContainerLogic',
+          props: {
+            ...programs,
+            sql: programsSql,
+          },
+        },
+      ],
+    },
```

Comment:
All RouteControlWrapper configs needed to be modified to contain the Explore components